lib/action_network_rest/signatures.rb

```diff
@@ -0,0 +1,23 @@
+module ActionNetworkRest
+  class Signatures < Base
```

Comment:
Should this instead be nestled under the petitions to match the URL routes, somehow?

I'm fine with this generally though, I feel like something more OO would be too heavy most likely.